src/views/splash/splash.jsx

```diff
@@ -215,11 +215,11 @@ var Splash = injectIntl(React.createClass({
                 <Box
                         key="curator_top_projects"
                         title={
-                            'Projects Curated by ' +
+                            formatMessage({id: 'splash.projectsCuratedBy', defaultMessage: 'Projects Curated by '}) +
```

Comment:
If this is in `splash/l10n.json`, then we actually don't need to include the `defaultMessage` – we could just add it by id.